gef.py

```diff
@@ -2808,6 +2819,14 @@ class X86(Architecture):
     _ptrsize = 4
     _endianness = Endianness.LITTLE_ENDIAN
 
+    # TODO: How do I show that this is a GefMemoryMapProvider?
+    @staticmethod
```

Comment:
this is just for testing. Feel free to test with an x86 binary, it should still work. We will remove this before merging, as we want to use the 'default' maps normally.